test/UnbrandedProjects/NoTest-TypeSpec/src/Generated/Internal/ClientUriBuilder.cs

```diff
@@ -42,11 +41,6 @@ public void AppendPath(string value, bool escape)
                 value = Uri.EscapeDataString(value);
             }
 
-            if (value[0] == PathSeparator)
-            {
-                value = value.Substring(1);
-            }
```

Comment:
this part might still be useful, because sometimes, we might have uris like this `//authentication/http/custom/valid` which leads to a request failure.